### DISCUSSION TITLE
Studio One FX and Buss channels not connecting

### DISCUSSION THREAD

**Discussion by baaccs** [apohl79/audiogridder#1145] [Help]
Hi Andreas,

So, I am back to learning Studio One after using Audiogridder in Pro Tools with no issues, and what is happening is when I open the plugin in the FX or Mix Buss it will not connect to the server and until I disable the plugin it just clicks and pops.  The regular input channels connect fine and work with no issues except for the Gui.  I decided I just use remote desktop for that issue.  Reading some on the other issues maybe I am assuming that the Gui issue is a Windows problem, correct?  I looked to see if anyone else running Studio reported that issue and I didn't find anything.  Just wondering am I missing something here because I am clueless for it to work good open the regular channels but the buss channels.

Thanks, Bruce

**Comment by apohl79** [comment]
If the GUI works on pro tools, it should also work in a different DAW. Could you create a short screen recording and provide the plugin and server logs, so I can see whats wrong and check the log output?

https://audiogridder.com/bug-report-diagnostics-locations/
